frontend/test/metabase/scenarios/sharing/alert/alert.cy.spec.js

```diff
@@ -207,3 +207,49 @@ describe("scenarios > alert", () => {
     });
   });
 });
+
+describeWithToken("scenarios > alert (EE)", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should validate for approved email domains", () => {
+    const email = "example@example.com";
+    const domain = "metabase.com";
```

Comment:
We're trying to refrain from using real TLDs in tests. Pleas use the reserved ones like `.test` or `.example`.

Reply:
Fixed